salt/covid19.sls

```diff
@@ -1,21 +1,23 @@
 {% from 'lib.sls' import create_user, set_firewall %}
+
 {{ set_firewall("PUBLIC_SSH") }}
 
+{% if salt['pillar.get']('python_apps:covid19admin:git:branch') or salt['pillar.get']('react_apps:covid19public:git:branch') %}
```

Comment:
To match the style below, we can do:

    {% if 'branch' in pillar.python_apps.covid19admin.git or 'branch' in pillar.react_apps.covid19public.git %}